docs/par2cron_completion_zsh.md

```
@@ -0,0 +1,52 @@
+#### Linux:
+
+	par2cron completion zsh > "${fpath[1]}/_par2cron"
+
+#### macOS:
+
+	par2cron completion zsh > $(brew --prefix)/share/zsh/site-functions/_par2cron
+
+You will need to start a new shell for this setup to take effect.
+
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)